src/main/java/nl/novi/techiteasy1121/models/Television.java

```diff
@@ -28,47 +30,66 @@ public class Television {
     private Integer originalStock;
```

Comment:
Overweeg om expliciete constructors in je modellen weg te laten, aangezien Spring Boot deze automatisch kan genereren. Dit zou de code eenvoudiger kunnen maken.